core/src/test/java/net/sourceforge/jnlp/cache/ResourceDownloaderTest.java

```diff
@@ -129,10 +129,10 @@ public void getUrlResponseCodeTestWorkingHeadRequest() throws Exception {
         redirectErr();
         try {
             File f = File.createTempFile(nameStub1, nameStub2);
-            int i = ResourceDownloader.getUrlResponseCode(testServer.getUrl(f.getName()), new HashMap<String, String>(), ResourceTracker.RequestMethods.HEAD);
+            int i = ResourceDownloader.getUrlResponseCode(testServer.getUrl(f.getName()), new HashMap<String, String>(), HttpMethod.HEAD);
             Assert.assertEquals(HttpURLConnection.HTTP_OK, i);
             f.delete();
-            i = ResourceDownloader.getUrlResponseCode(testServer.getUrl(f.getName()), new HashMap<String, String>(), ResourceTracker.RequestMethods.HEAD);
+            i = ResourceDownloader.getUrlResponseCode(testServer.getUrl(f.getName()), new HashMap<String, String>(), HttpMethod.HEAD);
```

Comment:
Are the concrete String generic definition for the HashMap really needed? Wouldn't' diamond operator with `new HashMap()<>` work?

Reply:
works without :) done